x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/discovery_admin_recon.json

```diff
@@ -0,0 +1,48 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Identifies instances of lower privilege accounts enumerating Administrator accounts or groups using built-in Windows tools.",
+  "index": [
+    "logs-endpoint.events.*",
+    "winlogbeat-*"
+  ],
+  "language": "eql",
+  "license": "Elastic License",
+  "name": "Enumeration of Administrator Accounts",
+  "query": "process where event.type in (\"start\", \"process_started\") and\n  (((process.name : \"net.exe\" or process.pe.original_file_name == \"net.exe\") or\n    ((process.name : \"net1.exe\" or process.pe.original_file_name == \"net1.exe\") and\n        not process.parent.name : \"net.exe\")) and\n   process.args : (\"group\", \"user\", \"localgroup\") and\n   process.args : (\"admin\", \"Domain Admins\", \"Remote Desktop Users\", \"Enterprise Admins\", \"Organization Management\") and\n   not process.args : \"/add\")\n\n   or\n\n  ((process.name : \"wmic.exe\" or process.pe.original_file_name == \"wmic.exe\") and\n     process.args : (\"group\", \"useraccount\"))\n",
```

Comment:
No comments here, this one should work right?
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/101856534-be25b080-3b22-11eb-9286-919af453da10.png" />
</p>

Reply:
Broke it out to make precedence more readable

```sql
process where event.type in ("start", "process_started") and
  (
      (
          (process.name : "net.exe" or process.pe.original_file_name == "net.exe") or
            (
              (process.name : "net1.exe" or process.pe.original_file_name == "net1.exe") and
                 not process.parent.name : "net.exe"
            )
      ) and
      process.args : ("group", "user", "localgroup") and
      process.args : ("admin", "Domain Admins", "Remote Desktop Users", "Enterprise Admins", "Organization Management") and
      not process.args : "/add"
  )

   or

  (
      (process.name : "wmic.exe" or process.pe.original_file_name == "wmic.exe") and
      process.args : ("group", "useraccount")
  )
```

Query looks good to me. Based on the location of the error, looks like it could also be an issue with case insensitive compare of multiple args, though I got the same error as above in siem-dev via EQL API search. Thoughts @rw-access

Reply:
https://github.com/elastic/detection-rules/issues/709